src/components/pages/Landing/RenderLandingPage.js

```diff
@@ -1,19 +1,75 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+// import { Link } from 'react-router-dom';
+import Button from '../../common/Button';
 
 function RenderLandingPage(props) {
   return (
-    <div>
-      <h1>Welcome to Labs Basic SPA</h1>
-      <div>
-        <p>
+    <div className="landing-container">
+      {/* <h1>Welcome to Labs Basic SPA</h1> */}
+      {/* <div> */}
+      {/* <p>
           This is an example of how we'd like for you to approach page/routable
           components.
-        </p>
-        <p>
-          <Link to="/example-list">Example List of Items</Link>
-        </p>
+        </p> */}
+
+      {/* Main text container */}
+      <div className="landing-main-container">
+        <div className="landing-main-text">
+          <h1>Coding is a superpower.</h1>
+          <h3>
+            {' '}
+            Through a love of teachin g and technology, we empower youth to
```

Comment:
Little typo on line 21, teaching has the g separated from the word.

Reply:
Thank you for reviewing my code. I fixed the typo that you mentioned above. 
https://github.com/BloomTech-Labs/coder-heroes-fe/pull/103/commits/c75a86413ed87d6f2f0ee1bb2bc87d56976ebf42